src/actions/player.js

```diff
@@ -4,9 +4,15 @@ const PREFIX = 'PLAYER';
 
 export const LOADING_PLAYER = `${PREFIX}/LOADING_PLAYER`;
 export const SET_CURRENT_PLAYER = `${PREFIX}/SET_CURRENT_PLAYER`;
+export const LOADING_PLAYERS = `${PREFIX}/LOADING_PLAYERS`;
+export const PLAYERS_LOADED = `${PREFIX}/PLAYERS_LOADED`;
+
+export const loadingPlayer = { type: LOADING_PLAYER };
```

Comment:
_Feel free to ignore me here since I don'y know much about general standards in React yet._
I wonder if, for consistency, it might make sense to have this and `loadingPlayers` as functions, that way we consistently always pass the result of a function call to `dispatch`. It'll also mean less future work if we need to add parameters to a state that didn't initially have them.

Reply:
We can talk more about this. I think it's less overhead to have it be a simple object instead of a function, but it's probably not a big deal. Also, this is the way we do it at work in all our projects 🤷‍♂

Reply:
I'd say go ahead and leave it as-is if it's the common way to do it ![bitmoji](https://sdk.bitmoji.com/render/panel/4f211e34-f0a8-4ad4-b294-4aadf9aa76b7-2ff83ea4-f4d8-43f8-9748-d3c45df5edf9-v1.png?transparent=1&palette=1&width=246)